Copy edit 5.16 guides (#2010)
sensu/sensu-docs: content/sensu-go/5.16/guides/clustering.md

---
title: "Run a Sensu cluster"
linkTitle: "Run a Sensu Cluster"

> description: "Clustering makes Sensu more highly available, reliable, and durable. It can help you cope with the loss of a backend node, prevent data loss, and distribute the network load of agents. Read the guide to configure a Sensu cluster."

Reviewer: ```suggestion
description: "Clustering increases Sensu's availability, reliability, and durability. It can help you cope with the loss of a backend node, prevent data loss, and distribute the network load of agents. Read the guide to configure a Sensu cluster."
```

> description: "Clustering increases Sensu's availability, reliability, and durability. It can help you cope with the loss of a backend node, prevent data loss, and distribute the network load of agents. Read the guide to configure a Sensu cluster."

Author: ```suggestion
description: "Clustering improves Sensu's availability, reliability, and durability. It can help you cope with the loss of a backend node, prevent data loss, and distribute the network load of agents. Read the guide to configure a Sensu cluster."
```

    parent: guides
---

- [Configure a cluster](#configure-a-cluster)
- [Manage and monitor clusters with sensuctl](#manage-and-monitor-clusters-with-sensuctl)
- [Manage cluster members](#add-a-cluster-member)
- [Cluster security](#cluster-security)
- [Use an external etcd cluster](#use-an-external-etcd-cluster)
- [Troubleshoot clusters](#troubleshoot-clusters)

A Sensu cluster is a group of [at least three][1] sensu-backend nodes, each connected to a shared etcd cluster, using Sensu's embedded etcd or an external etcd cluster.
Creating a Sensu cluster ultimately configures an [etcd cluster][2].

> Clustering increases Sensu's availability, reliability, and durability.

Author: ```suggestion
Clustering improves Sensu's availability, reliability, and durability.
```

> Clustering is important to make Sensu more highly available, reliable, and durable.

Reviewer: ```suggestion
Clustering increases Sensu's availability, reliability, and durability.
```
